CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/ConversationBox.java

```diff
@@ -266,24 +266,20 @@ public ConversationBox(final Conversation conversation) {
         // This code below enables the graph in sync with the search results.
         searchBubbleTextField.textProperty().addListener((ov, oldValue, newValue) -> {
             final Graph graph = conversation.getGraphUpdateManager().getActiveGraph();
-            ReadableGraph readableGraph = graph.getReadableGraph();
-            try {
-                final List<ConversationContributionProvider> compatibleContributionProviders;
-                compatibleContributionProviders = ConversationContributionProvider.getCompatibleProviders(readableGraph);
-                for (final ConversationMessage message1 : messages) {
-                    message1.getAllContributions().clear();
+            final ReadableGraph readableGraph = graph.getReadableGraph();
+            try (readableGraph) {
```

Comment:
You can move the declaration of `readableGraph` into the try-with-resources. i.e.
`try (final ReadableGraph readableGraph = graph.getReadableGraph)`